apps/prairielearn/src/ee/pages/instructorAiGenerateQuestion/instructorAiGenerateQuestion.html.ts

```diff
@@ -36,109 +39,337 @@ const examplePrompts = [
+                          <div class="form-text form-muted">
+                            <em>
+                              Example: students should enter the solution using a decimal number.
+                              The answer should be in seconds.
+                            </em>
+                          </div>
+                          <label for="user-prompt-llm-grading">
+                            How do we determine if a solution is correct/which answer choice is
+                            correct?
+                          </label>
+                          <textarea
+                            name="prompt_grading"
+                            id="user-prompt-llm-grading"
+                            class="form-control"
+                          ></textarea>
+                          <div class="form-text form-muted">
+                            <em>
+                              Example: the answer is computed as sqrt(2 * h / g) where g = 9.81
+                              m/s^2
+                            </em>
+                          </div>
+                        </div>
+                        <button class="btn btn-primary">
+                          <span
+                            class="spinner-grow spinner-grow-sm d-none"
+                            role="status"
+                            aria-hidden="true"
+                            data-loading-class-remove="d-none"
+                          >
+                          </span>
+                          Create question
+                        </button>
+                      </form>
+                      Or choose a question from our list of example prompts:
+                      <select
+                        id="user-prompt-example"
+                        onchange="setPromptToExample()"
+                        class="custom-select"
+                      >
+                        <option value=""></option>
+                        ${examplePrompts.map(
+                          (question) =>
+                            html`<option
+                              value="${question.id}"
+                              data-prompt-general="${question.promptGeneral}"
+                              data-prompt-user-input="${question.promptUserInput}"
+                              data-prompt-grading="${question.promptGrading}"
+                            >
+                              ${question.id}
+                            </option>`,
+                        )}
+                      </select>
+                      <div id="generation-results"></div>
+                    </div>`
+                  : html`<div class="container">
+                        <div class="row justify-content-center">
+                          <div class="col">
+                            ${prompts
+                              .filter((x) => {
+                                return x.prompt_type !== 'auto_revision';
```

Comment:
right now, there's not a publicly available way to check error detection and regeneration- should we add a separate page for it?